### PR TITLE
[Enhancement] Zonemap compatible with previous version-0.12

### PR DESCRIPTION
**Pull request by xinghuayu007**
## Proposed changes

Doris version-0.12 V1 segment  only supports creating zone map index for key columns. Doris version-0.13 supports creating zone map index for key columns and duplicate model columns. Lastest version supports creating zone map for key columns in AGG_KEYS or all columns in UNIQUE_KEYS or DUP_KEYS. In V1 Segment file, the tablet meta contains zone map index. But in V2 Segment file, it does not.

When upgrade Doris from version-0.12 to version-0.13, we found the memory is obviousely increasing in table with V1 segement file. Because a lots value columns create zone map index. Therefore, it is better to be compatible with version-0.12. The config `enable_value_column_zonemap` is default false. If it is need to add zone map index for value colums, you can set it true.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] If these changes need document changes, I have updated the document
- [x] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by morningman — DISMISSED** [review]
LGTM

File: be/src/common/config.h [hunk 1]
```diff
@@ -598,6 +598,8 @@ CONF_mInt32(zone_map_row_num_threshold, "20");
 //    Trace = 6
 CONF_Int32(aws_log_level, "3");
 
+CONF_mBool(enable_value_column_zonemap, "false");
```
Comment (morningman) [inline be/src/common/config.h:601]: I think we can just remove this config, and just not add zone map for v1's value column, to make things simple.
Do we have any reason to add zonemap for v1's value column now?

Comment (xinghuayu007) [reply be/src/common/config.h]: ok

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**